src/utils.ts

```diff
@@ -30,6 +31,14 @@ import { CliInitDbOptions, JobManagerOptions } from './definitions'
 import { JobsManager } from './jobs-manager'
 import { IpfsProvider } from './providers/ipfs'
 
+export function bn (v: any): BigNumber {
```

Comment:
Not a big fan. See https://github.com/rsksmart/rif-marketplace-cache/pull/254#discussion_r469495298

Reply:
fixed